test/spec/modules/gridBidAdapter_spec.js

```diff
@@ -118,6 +118,24 @@ describe('TheMediaGrid Adapter', function () {
       }
     ];
 
+    it('should be content categories and genre', function () {
+      const site = {
+        cat: ['IAB2'],
+        pagecat: ['IAB2-2'],
+        content: {
+          genre: 'Adventure'
+        }
+      };
+
+      const getConfigStub = sinon.stub(config, 'getConfig').callsFake(
+        arg => arg === 'ortb2.site' ? site : null);
+      const request = spec.buildRequests([bidRequests[0]], bidderRequest);
+      const payload = parseRequest(request.data);
+      expect(payload.site.cat).to.deep.equal([...site.cat, ...site.pagecat]);
```

Comment:
Please fix your circleci test output, the number of failures is "too large to display in browser"

Reply:
Hello, we have fixed circleci tests